### PR TITLE
[IoTCentral]CLI documentation improvements

### PR DESCRIPTION
**Pull request by kawayiboy**
**Description**  
[IoTCentral]CLI documentation improvements

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by kawayiboy** [comment]
@zhoxing-ms @MyronFanQiu @Juliehzl , Please take a look at this PR changing some help text. And for the build test error:
test_network_security_partner_provider (azure.cli.command_modules.network.tests.latest.test_network_commands.NetworkSecurityPartnerProviderScenarioTest)
test_network_virtual_appliance (azure.cli.command_modules.network.tests.latest.test_network_commands.NetworkVirtualApplianceScenarioTest)

they seems are not introduced by my change but introduced by network command modules. Please help on that part. Thanks

**Comment by yonzhan** [comment]
IoTCentral

File: src/azure-cli/azure/cli/command_modules/iot/_help.py [hunk 1]
```diff
@@ -768,60 +768,80 @@
...
 helps['iot central app delete'] = """
 type: command
 short-summary: Delete an IoT Central application.
+long-summary: |
+    Permanently delete your IoT Central app.
+    -Warning- This operation is irreversible. Once you've deleted your app, you cannot restore it.
 examples:
-  - name: Delete an IoT Central application.
-    text: az iot central app delete --name MyIoTCentralApplication --resource-group MyResourceGroup
+  - name: Deletes an IoT Central app called 'my-first-iot-app'.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/iot/_help.py:806]: Please use the first person voice~
```suggestion
  - name: Delete an IoT Central app called 'my-first-iot-app'.
```

**Comment by zhoxing-ms** [comment]
@kawayiboy Could you please sync the code from the remote dev branch?
Then please rerun the test of `test_network_virtual_appliance` in live mode, and submit the yaml file which is re-recorded.
( Run `azdev test test_network_virtual_appliance --live` locally)

**Comment by kawayiboy** [comment]
@zhoxing-ms ,  I fixed the build

File: src/azure-cli/azure/cli/command_modules/iot/_params.py [hunk 2]
```diff
@@ -276,14 +276,22 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
...
-        c.argument('sku', arg_type=get_enum_type(AppSku),
-                   help='Pricing tier for IoT Central applications. Default value is ST2.')
+                   help='Where your app\'s info and resources are stored. We will default to the location'
+                        ' of the target resource group. See documentation for a full list of supported locations.')
+        c.argument('sku', arg_type=get_enum_type(AppSku), options_list=['--sku', '-p'],
+                   help='Pricing plan for IoT Central application.')
         c.argument('subdomain', options_list=['--subdomain', '-s'],
-                   help='Subdomain for the IoT Central URL. Each application must have a unique subdomain.')
+                   help='Enter a unique URL. Your app will be accessible via https://<subdomain>.azureiotcentral.com/.'
+                   ' Avoid special characters - instead, use lower case letters (a-z), numbers (0-9), and dashes (-).')
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/iot/_params.py:291]:
```suggestion
                   ' Avoid special characters `-` instead, use lower case letters (a-z), numbers (0-9), and dashes (-).')
```

**Comment by kawayiboy** [comment]
@zhoxing-ms @MyronFanQiu Could you help merge this PR? thanks

**Pull request merged**